src/commands/force/org/beta/create.ts

```diff
@@ -250,7 +262,104 @@ export class Create extends SfdxCommand {
     }
   }
 
-  private createScratchOrg(): void {
-    //
+  private async saveAuthInfo(username: string): Promise<void> {
+    if (this.flags.setalias) {
+      const alias = await Aliases.create(Aliases.getDefaultOptions());
+      alias.set(this.flags.setalias, username);
+      const result = await alias.write();
+      this.logger.debug('Set Alias: %s result: %s', this.flags.setalias, result);
+    }
+    if (this.flags.setdefaultusername) {
+      const globalConfig: Config = this.configAggregator.getGlobalConfig();
+      globalConfig.set(Config.DEFAULT_USERNAME, username);
+      const result = await globalConfig.write();
+      this.logger.debug('Set defaultUsername: %s result: %s', this.flags.setdefaultusername, result);
+    }
+  }
+  private async createScratchOrg(): Promise<ScrathcOrgProcessObject> {
+    this.logger.debug('OK, will do scratch org creation');
+    if (!this.hubOrg) {
+      throw SfdxError.create('@salesforce/plugin-org', 'org', 'RequiresDevhubUsernameError');
+    }
+    // Ensure we have an org config input source.
+    if (!this.flags.definitionjson && !this.flags.definitionfile && Object.keys(this.varargs).length === 0) {
+      this.ux.log('--------------------');
+      this.ux.log('a definition or varargs is required');
+      this.logger.error('a definition or varargs is required');
+      throw new SfdxError(messages.getMessage('cliForceCreateNoConfig'));
```

Comment:
NIT: this error name is reminiscent of toolbelt. We could remove `cli` and `force` and `create` from it now that we know it's scoped to `plugin-org`